src/Element.cc

```diff
@@ -279,7 +279,7 @@ ElementPtr Element::Clone(sdf::Errors &_errors) const
   if (this->dataPtr->value)
   {
     clone->dataPtr->value = this->dataPtr->value->Clone();
-    SDF_ASSERT(clone->dataPtr->value->SetParentElement(clone),
+    SDF_ASSERT(clone->dataPtr->value->SetParentElementNoReparse(clone),
```

Comment:
with the context that `Param::Reparse` was added in https://github.com/gazebosim/sdformat/pull/589 to support parsing pose values based on the parent element's `@degrees` attribute value, I think it may be fine to clone Param values without reparsing.